GameStore.Infrastructure/Services/GameService.cs

```diff
@@ -85,14 +85,20 @@ public async Task<IEnumerable<GameModel>> GetByPlatformTypeAsync(int platformId)
 
     private async Task AddGenresAndPlatforms(Game game, List<int> genreIds, List<int> platformIds)
     {
-        foreach (var id in genreIds)
+        if (genreIds != null)
         {
-            game.Genres.Add(await _unitOfWork.GenreRepository.GetByIdAsync(id));
+            foreach (var id in genreIds)
+            {
+                game.Genres.Add(await _unitOfWork.GenreRepository.GetByIdAsync(id));
+            }
         }
 
-        foreach (var id in platformIds)
+        if (platformIds != null)
```

Comment:
Try to get rid of this ugly null checks. Tip: use `new()` inside `GameCreateModel`